components/layout/user-nav.tsx

```diff
@@ -13,21 +13,22 @@ import {
 } from '@/components/ui/dropdown-menu';
 import { useSession } from 'next-auth/react';
 import { extractInitials } from '@/lib/utils';
+import { ACCOUNT_TYPES } from '@/constants/generic';
 
 export function UserNav() {
   const { data: session } = useSession();
 
   return (
     <DropdownMenu>
       <DropdownMenuTrigger asChild>
-        <Button variant="ghost" className="relative h-8 w-8 rounded-full">
+        <Button variant='secondary' className="relative h-8 w-8 rounded-full text-white">
           <Avatar className="h-8 w-8">
             {/* <AvatarImage
               src={session.user?.image ?? ''}
               alt={session.user?.name ?? ''}
             /> */}
             <AvatarFallback>
-              {extractInitials(session?.user?.fullName ?? '')}
+              {extractInitials(session?.user?.email ?? '')}
```

Comment:
please revert this back to full name. the extract initials function is a function that extract your initials if your fullname is provided, for example Shereef Habib would return "SH". if you remove the full name and provide the email there it means we wouldnt have the functionality working properly

Reply:
It is not returning no data was return as fullName from the session